compiler_opt/rl/trainer.py

```diff
@@ -50,6 +50,7 @@ def __init__(
       # Params for summaries and logging
       checkpoint_interval=10000,
       log_interval=100,
+      summary_log_interval=100,
       summary_interval=1000,
```

Comment:
nit: can you change it to summary_export_interval to be more clear?

Reply:
sorry for the confusion...github is not clear in terms of what line i choose -_-!

i mean to have summary_log_interval for the ones you added, and rename summary_interval to be summary_export_interval to make these two flags more clear

Reply:
Should be fixed now. I should probably also pay more attention to what line a comment indicates (since it's the bottom one), and also think about what I'm doing a little bit more. :p